### ISSUE TITLE
Unhandled exception (#aba558d8)

### ISSUE THREAD

**Issue by sqlmapreporter** [sqlmapproject/sqlmap#3059]
```
Running version: 1.2.4#stable
Python version: 2.7.10
Operating system: Darwin-17.5.0-x86_64-i386-64bit
Command line: /usr/local/bin/sqlmap -u ************************************************
Technique: None
Back-end DBMS: None
```
```
Traceback (most recent call last):
  File "sqlmap", line 157, in main
    start()
  File "Cellar/sqlmap/1.2.4/libexec/lib/controller/controller.py", line 398, in start
    if not checkConnection(suppressOutput=conf.forms) or not checkString() or not checkRegexp():
  File "Cellar/sqlmap/1.2.4/libexec/lib/controller/checks.py", line 1557, in checkConnection
    if conf.hostname in (threadData.lastRedirectURL[1] or "") and threadData.lastRedirectURL[1].startswith("https://"):
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 39: ordinal not in range(128)
```

**Comment by stamparm** [comment]
Duplicate of #3022 (fixed)

**Comment by neo1230** [comment]
how do I fix error #3022 in mysql

**Comment by stamparm** [comment]
@neo1230 just get the latest revision from this repository (i.e. update sqlmap)

Latest one is `1.2.4.18` which is greater than `1.2.4` which is having problems with this bug you reported
